prosilica_camera/src/nodes/prosilica_nodelet.cpp

```diff
@@ -253,6 +254,21 @@ class ProsilicaNodelet : public nodelet::Nodelet
                 {
                     state_info_ = "Trying to load camera with ipaddress: " + ip_address_;
                     NODELET_INFO("%s", state_info_.c_str());
+                    /* Resolve hostname */
+                    struct hostent *he;
+                    if ((he = gethostbyname(ip_address_.c_str())) != NULL)
+                    {
+                      struct in_addr **addr_list
+                        = (struct in_addr **)he->h_addr_list;
```

Comment:
Why do you have to cast it to an `struct in_addr` rather than leave it as the normal `char **` that it normally returns? I'm referencing this document:

http://www.gnu.org/software/libc/manual/html_node/Host-Names.html

Also, if you must cast it, please use the c++ style `static_cast`.